suite-native/graph/src/components/Graph.tsx

```diff
@@ -33,13 +70,37 @@ export const Graph = ({ points = [] }: GraphProps) => {
 
     return (
         <Box style={applyStyle(graphWrapperStyle)}>
-            <LineGraph
-                style={applyStyle(graphStyle)}
-                points={graphPoints}
-                color="#00854D"
-                animated
-                enablePanGesture
-            />
+            {nonZeroBalance ? (
+                <LineGraph
+                    style={applyStyle(graphStyle)}
+                    points={graphPoints}
+                    color="#00854D"
+                    animated
+                    enablePanGesture
+                    TopAxisLabel={
```

Comment:
these two should be wrapped in `useMemo` and you can even include that `if` in that `useMemo`

Reply:
90a58371ff675ff2f27b4477b9081f78bfacd50b